src/LibraryUtilities.ts

```diff
@@ -116,7 +114,6 @@ export class ItemData {
         }
 
         // Get keywords from typeListNode and push them into itemData
-        let keywords = typeListNode.keywords.split(",");
```

Comment:
why do we remove this line?

Reply:
@QilongTang This variable is not used. The operations below this statement refers to keywords property of class ItemData class (`this.keywords`)